run_pipeline.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+PROJECT_ROOT_DIR=$(dirname "$(realpath "${BASH_SOURCE[0]}")")
```

Comment:
this is basically a copy of tempatlize.sh
can we try to pull the common parts into a script we include in both templatlize.sh and run_pipeline.sh?

Reply:
Sure can do. I somehow feared having a single script can be quite complicated, but I'll give it a shot.